test/test_other.py

```diff
@@ -1307,6 +1307,37 @@ def test_export_all(self):
     self.emcc('lib.c', ['-Oz', '-sEXPORT_ALL', '-sLINKABLE', '--pre-js', 'main.js'], output_filename='a.out.js')
     self.assertContained('libf1\nlibf2\n', self.run_js('a.out.js'))
 
+  def test_modularize_export_keepalive(self):
```

Comment:
I think you can drop the modularize part here... just testing with the default config should be enough.